content/customize-gnome.md

```diff
@@ -44,6 +44,8 @@ This extension allows for the desktop to be themed in further ways than default.
 
 ![dash-to-dock](/images/customize-gnome/dash-to-dock.png)
 
+ >**Note:** Starting with Pop!\_OS 22.04 there is a custom dock forked from Dask to Dock with additional options in Settings -> Desktop -> Dock.
```

Comment:
s/Dask/Dash

Also is that extension `Dash-To-Dock` or `Dash To Dock`?

Reply:
Also needs to mention that you need to disable Cosmic-Dock in extensions or odd things happen.

Reply:
Seems to be Dash to Dock: 

https://extensions.gnome.org/extension/307/dash-to-dock/